### PR TITLE
Enable JWT settings for Open edX installs CRI-173

### PR DESCRIPTION
**Pull request by nedbat**
This also splits COMMON_SANDBOX_BUILD into its two components: SANDBOX_CONFIG and CONFIGURE_JWTS.

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/openedx_native.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@
     COMMON_ENABLE_AWS_ROLE: false
     ecommerce_create_demo_data: true
     credentials_create_demo_data: true
+    # We are not building a sandbox, but this turns on jwt configuration:
+    COMMON_SANDBOX_BUILD: true
```
Comment (adzuci) [inline playbooks/openedx_native.yml]: Could we rename this and split it out from the sandbox CNAME bits rather than merge this as is?

Comment (nedbat) [reply playbooks/openedx_native.yml]: I am fine with splitting it, but could use some guidance.

**Review by adzuci — APPROVED** [review]
LGTM

**Comment by adzuci** [comment]
@matthugs Will this impact existing registrar sandboxes?

**Review by matthugs — APPROVED** [review]
I don't anticipate an adverse impact on master's sandbox builds, since ansible-provision.sh is also our main entry point into this labyrinth, and Ned's addressed that. Thanks for the tag @adzuci

**Pull request merged**